src/main/scala/com/gu/playdeveloperapi/Conversion.scala

```diff
@@ -0,0 +1,37 @@
+package com.gu.playdeveloperapi
+
+import com.gu.playdeveloperapi.PlayDeveloperApi.PlayDeveloperApi.{ Track, TracksResponse }
+
+import scala.util.{ Failure, Success, Try }
+
+object Conversion {
+
+  case class Build(versionName: String)
+  case class AndroidLiveAppVersions(currentBeta: Build, currentProduction: Build)
+
+  case object PlayDeveloperApiConversionException extends Throwable
+
+  def searchForTrack(allTracks: List[Track], trackName: String): Option[Build] = {
```

Comment:
What is a track?

Reply:
In Google Play, 'tracks' represent the different possible release channels (e.g. `production`, `beta`, `alpha`); I guess it would be roughly equivalent to our concept of a `Stage` (`PROD` or `CODE`). Users who are associated with different tracks will receive different app versions via the Play Store.